### PR TITLE
Fix: IO Exception on closed file

### PR DESCRIPTION
**Pull request by najma-kunju**

### CODE REVIEW

**Comment by najma-kunju** [comment]
About this PR:

While running a query. with RESULT_STORE_TYPE as file, the execution completes but shows below error while displaying result.

**I/O exception opening closed file**

Screenshot 2023-04-26 at 12 26 31 PM


So as a fix, created a PR.

**Comment by najma-kunju** [comment]
Hi @jczhong84,
Can you please have a look at this PR too?

File: querybook/server/lib/result_store/stores/file_store.py [hunk 1]
```diff
@@ -61,9 +62,14 @@ def start(self):
...
     def get_csv_iter(self, number_of_lines: Optional[int]):
-        with open(self.uri) as result_file:
-            reader = csv.reader(result_file)
-            return islice(reader, number_of_lines)
+        # with open(self.uri) as result_file:
+        #     reader = csv.reader(result_file)
+        #     return islice(reader, number_of_lines)
+
+        # Execution completes but was showing error "I/O exception opening closed file" while displaying result. 
+        # Commenting the above and adding the below as fix.
```
Comment (jczhong84) [inline querybook/server/lib/result_store/stores/file_store.py:70]: please remove all those commented-out lines, they not needed with the new implementation.

**Comment by jczhong84** [comment]
Thanks for the fix!

please fix the lint errors

File: querybook/server/lib/result_store/stores/file_store.py (hunk 1 above)
Comment (najma-kunju) [reply querybook/server/lib/result_store/stores/file_store.py]: done

Comment (najma-kunju) [reply querybook/server/lib/result_store/stores/file_store.py]: Hi @jczhong84,
I have fixed the lint errors too.

**Comment by jczhong84** [comment]
@najma-kunju there is still lint failure

**Comment by najma-kunju** [comment]
@jczhong84, have fixed the current lint error

**Review by jczhong84 — APPROVED** [review]

**Pull request merged**